Microsoft.Azure.CosmosRepository/src/DefaultRepository.cs

```diff
@@ -279,6 +279,39 @@ public async ValueTask<bool> ExistsAsync(Expression<Func<TItem, bool>> predicate
             return count > 0;
         }
 
+        public async ValueTask<IPage<TItem>> ScrollAsync(int lastPage = 0,

```

Comment:
I'm curious how this makes @roji feel? What do you think about this API and implementation?

Reply:
Yes very keen to hear some feedback 😃